nautobot/core/api/views.py

```diff
@@ -747,15 +746,18 @@ def format_and_remove_hidden_menu(self, request, data, hide_restricted_ui):
             Output:
             {"Devices": "data value"}
         """
-        if isinstance(data, dict):
```

Comment:
Why is the `isinstance` check no longer needed? Is `data` now always required/guaranteed to be a dict? (If so, please update the docstring above accordingly.)

Reply:
Because I set `the'return_value`' instead of recalling the function on line `754`, and that's the only place where data may be anything other than `dict`.